ubtue_Brill.js

```diff
@@ -33,69 +33,220 @@
 	***** END LICENSE BLOCK *****
 */
 
-
 function detectWeb(doc, url) {
-	if (url.match(/article-.+\.xml$/)) {
-		return "journalArticle";
-	} else if (url.match(/issue-\d+(-\d+)?\.xml$/)) {
-		return "multiple";
- 	}
-}
-
-function getSearchResults(doc) {
-	let items = {};
-    let found = false;
-    let links = ZU.xpath(doc, '//a[contains(@class, "c-Typography--title")]');
-	let text = ZU.xpath(doc, '//a[contains(@class, "c-Typography--title")]/span')
-	for (let i = 0; i < links.length; ++i) {
-		let href = links[i].href;
-		let title = ZU.trimInternal(text[i].textContent);
-        if (!href || !title) continue;
-        if (!href.match(/article-.+\.xml$/))
-            continue;
-
-		found = true;
-		items[href] = title;
-	}
-	return found ? items : false;
+	//for zotaut rss procedure we do not need multiple downloads
+	return "journalArticle";
```

Comment:
I think simply removing multiple handling is a very idea that might lead to numerous problems. 
The URLs referenced https://github.com/ubtue/DatenProbleme/issues/1275 could no longer be tested